simplekv/cache.py

```diff
@@ -110,6 +110,26 @@ def open(self, key):
             # cache error, ignore completely and return from backend
             return self._dstore.open(key)
 
+    def _copy(self, source, dest):
+        """Implementation of :meth:`~simplekv.CopyMoveMixin.copy`.
+        
+        Copies the data in the backing store and removes the destination key from the cache,
+         in case it was already populated.
+         """
+        k = self._dstore._copy(source, dest)
+        self.cache.delete(dest)
```

Comment:
No, because if the backing store has a real copy implementation, which avoids the roundtrip store->python instance->store we want to avoid this roundtrip, instead of doing cache-store -> python instance -> backing store.